functions/supplier/file-supplier/src/test/java/org/springframework/cloud/fn/supplier/file/DefaultFileSupplierTests.java

```diff
@@ -104,6 +106,10 @@ public void testBasicFlow() throws IOException {
 		assertThat(metadataStoreContent.get(0)).startsWith("local-file-system-metadata-");
 		assertThat(metadataStoreContent.get(0)).endsWith("first.file");
 		assertThat(metadataStoreContent.get(1)).endsWith("test.file");
+
+		assertThat(TestUtils.getPropertyValue(this.fileMessageSource, "watchEvents",
```

Comment:
Do you mind adding a short comment describing why this is here, something like "the integration customizer in the test app adds watch events to the message source blah blah blah" :)